pages/index.page.tsx

```diff
@@ -417,7 +417,7 @@ const Home = (props: any) => {
               <div>
                 <Link
                   href={`/blog/posts/${blogPosts[0].slug}`}
-                  className='block w-full lg:w-1/2 rounded border-2 bg-primary text-white  h-[40px] text-center pt-1 semi-bold flex items-center justify-center'
+                  className='block w-full lg:w-1/2 rounded border-2 bg-primary text-white hover:bg-blue-700 transition-all duration-500 ease-in-out h-[40px] text-center pt-1 semi-bold flex items-center justify-center'
```

Comment:
```suggestion
                  className=' w-full lg:w-1/2 rounded border-2 bg-primary text-white hover:bg-blue-700 transition-all duration-500 ease-in-out h-[40px] text-center pt-1 semi-bold flex items-center justify-center'
```

Reply:
suggesting these changes to remove vs code warnings coming from using same tailwind utility classes

Reply:
I don't think it will give any errors due to - using same tailwind utility classes but anyway it is obsolete so should be removed.